tests/devices/test_dae.py

```diff
@@ -12,33 +12,33 @@
 from ophyd_async.testing import get_mock_put, set_mock_value
 
 from ibex_bluesky_core.devices import compress_and_hex, dehex_and_decompress
-from ibex_bluesky_core.devices.dae import convert_xml_to_names_and_values, set_value_in_dae_xml
-from ibex_bluesky_core.devices.dae.dae import Dae, RunstateEnum
-from ibex_bluesky_core.devices.dae.dae_controls import BeginRunExBits
-from ibex_bluesky_core.devices.dae.dae_period_settings import (
+from ibex_bluesky_core.devices.dae import (
+    Dae,
     DaePeriodSettings,
     DaePeriodSettingsData,
-    PeriodSource,
-    PeriodType,
-    SinglePeriodSettings,
-    _convert_period_settings_to_xml,
-)
-from ibex_bluesky_core.devices.dae.dae_settings import (
     DaeSettings,
     DaeSettingsData,
-    TimingSource,
-)
-from ibex_bluesky_core.devices.dae.dae_spectra import DaeSpectra
-from ibex_bluesky_core.devices.dae.dae_tcb_settings import (
-    CalculationMethod,
+    DaeSpectra,
     DaeTCBSettings,
     DaeTCBSettingsData,
+    DaeTimingSource,
+    PeriodSource,
+    PeriodType,
+    RunstateEnum,
+    SinglePeriodSettings,
+    TCBCalculationMethod,
+    TCBTimeUnit,
     TimeRegime,
     TimeRegimeMode,
     TimeRegimeRow,
-    TimeUnit,
-    _convert_tcb_settings_to_xml,
 )
+from ibex_bluesky_core.devices.dae._helpers import (
+    _convert_xml_to_names_and_values,
+    _set_value_in_dae_xml,
+)
+from src.ibex_bluesky_core.devices.dae import BeginRunExBits
+from src.ibex_bluesky_core.devices.dae._period_settings import _convert_period_settings_to_xml
+from src.ibex_bluesky_core.devices.dae._tcb_settings import _convert_tcb_settings_to_xml
```

Comment:
I think this import happens to work, but can we prefer

```suggestion
from ibex_bluesky_core.devices.dae import BeginRunExBits
from ibex_bluesky_core.devices.dae._period_settings import _convert_period_settings_to_xml
from ibex_bluesky_core.devices.dae._tcb_settings import _convert_tcb_settings_to_xml
```

please?